docs/getting-started.md

```diff
@@ -282,3 +316,27 @@ webserver.host` parameter from `127.0.0.1` to your AWS instance's private IP4 ad
 You'll also need to open port 8090 in the security group of your AWS instance to access Gravitino. To access Hive you need to open port 10000 in the security group.
 
 After completing these steps, you should be able to access the Gravitino REST interface from either the command line or a web browser on your local computer. You can also connect to Hive via DBeaver or any other database IDE.
+
+## Conclusion and Next Steps
+
+Congratulations! You've successfully set up Gravitino on your chosen platform. Here's a quick summary of key points and some next steps:
+
+### Next Steps:
+
```

Comment:
Would be nice to link each of these sections to other documentation.

Reply:
Could you give me an example ?

Reply:
For example link to https://datastrato.ai/docs in step 1.